### DISCUSSION TITLE
How to create image file on Windows PC?

### DISCUSSION THREAD

**Discussion by ezpc98** [johang/sd-card-images#107] [Q&A]
I'm new to linux and looking to set up my first SBC. On build page, instructions are to zcat two compressed image files (boot & debian bookworm image) into a new image file and then copy onto SD for booting the board.

```
$ zcat boot-raspberrypi_3b.bin.gz debian-buster-arm64-XXXXXX.bin.gz > sd-card.img
# dd if=sd-card.img of=/dev/sdXXX
```

I imagine it assumes one is already working on a linux distro. How can I build the same final image file on Windows PC?

I know it would need to be flashed onto SD card using Etcher tool but in need of help to get started with constructing the image file first.

**Comment by hw-schrauber** [comment]
@ezpc98
I struggled also with this problem and find out by myself this way on my Windows10 System:

1. Activate "Windows-Subsystem für Linux (WSL)"
2. Download the boot and system image to a directory to your download directory on the windows system. (c:\downloads for example)
3. run: #  zcat /mnt/c/Downloads/boot-orange_pi_r1.bin.gz /mnt/c/Downloads/debian-bullseye-armhf-jeush4.bin.gz > /mnt/c/Downloads/debian-bullseye-armhf-jeush4.img
4. have look to your download directory and you find the "debian-bullseye-armhf-jeush4.img"
5. start on win "Raspberry Pi Imager" (or any other Imager) and write the "debian-bullseye-armhf-jeush4.img" to your SD-card
6. next steps as known

Change "debian-bullseye-armhf-jeush4" to the System you want. 

Good luck!
